pyensign/ensign.py

```diff
@@ -117,15 +127,13 @@ def next():
             for event in events:
                 yield event.proto()
 
-        errors = []
-        async for rep in self.client.publish(topic_id, next()):
-            if isinstance(rep, ensign_pb2.Ack):
-                continue
-            elif isinstance(rep, ensign_pb2.Nack):
-                errors.append(rep)
-            else:
-                raise EnsignTypeError(f"unexpected response type: {type(rep)}")
-        return errors
+        await self.client.publish(
```

Comment:
Changed to utilize callbacks instead of blocking on publish, which I think matches better with the Go SDK right?

Reply:
It does yes - I think this is a good idea. We may want to consider a high level callback mechanism for subscribe events as well.